recipes/ucc/meta.yaml

```diff
@@ -0,0 +1,50 @@
+package:
+  name: ucc
+  version: v1.2.0
+
+source:
+  url: https://github.com/openucx/ucc/archive/refs/tags/v1.2.0.tar.gz
+  sha256: c1552797600835c0cf401b82dc89c4d27d5717f4fb805d41daca8e19f65e509d
+  patches:
+    - 0001-EC-CUDA-fix-build-with-12.2-and-older-arch-847.patch  # [(cuda_compiler_version or "None").startswith("12")]
+
+build:
```

Comment:
Please add appropriate run_exports for this package. (What versions of this package are ABI compatible?)

Reply:
Yup, see https://github.com/conda-forge/staged-recipes/pull/25886/commits/5ce134f6ddf0d4837815df6b998c5e110639fbb6#r1578819440